templates/docs/examples/patterns/navigation/sliding-dark.html

```diff
@@ -0,0 +1,118 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Navigation / Sliding dark{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
+
+{% block content %}
+<header id="navigation" class="p-navigation--sliding is-dark">
+  <div class="p-navigation__row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="#">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="" >
+          </div>
+          <span class="p-navigation__logo-title">Ubuntu</span>
+        </a>
+      </div>
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item">
+          <button class="js-menu-button p-navigation__link">Menu</button>
+        </li>
+      </ul>
+    </div>
+    <nav class="p-navigation__nav">
+      <div class="p-navigation__plane" id="main-nav" aria-label="Example main" data-level="0">
```

Comment:
Is this "__plane" needed? This seems to barely have any CSS on it. Also attributes seem to be moved from `nav` element (id, aria-label - these should be on `nav` I believe), and is `data-level` used for anything?

Reply:
So I guess it's not an HTML you @ClementChaumel created for this CSS, but HTML from #4714 @pedoch implementation? If so, this needs to be cleaned up, because I guess we don't really know what is the difference between this HTML and current navigation HTML (maybe it's just these `__plane` wrappers), but are they even needed in your implementation?

Reply:
Right, I removed it!

Reply:
Doesn't look removed from here ;) I guess you need to update all of examples.

BTW, not sure how much code these examples share (at least dark/light should be practically the same with one class name different), but it would be worth extracting common parts to some partial.